.travis.yml

```diff
@@ -4,7 +4,7 @@ before_script:
   - pip install gen3utils --upgrade
```

Comment:
let's install from your branch and make changes to some ETL mappings for testing. I'd say 3 files, 1 valid changes, 2 invalid changes. otherwise looks good